apps/server/src/modules/tool/controller/dto/response/oauth2-tool-config.response.ts

```diff
@@ -0,0 +1,37 @@
+import { ToolConfigType } from '@src/modules/tool/interface/tool-config-type.enum';
+import { ExternalToolConfigResponse } from '@src/modules/tool/controller/dto/response/external-tool-config.response';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class Oauth2ToolConfigResponse extends ExternalToolConfigResponse {
```

Comment:
Why not extends from BasicToolConfigResponse

Reply:
see above